### ISSUE TITLE
Question About Bootstrap Table + Extensions‏

### ISSUE THREAD

**Issue by SteveyPugs** [wenzhixin/bootstrap-table#1666]
Love your package. I had a few questions.
- For the bootstrap table defaults is there a way to apply to styles to just the headers via js
- For the extension filter control is there a way to apply styles (such as in input-sm so its smaller)
- For the extension filter control is there a way to auto set one of those filters on page load. For example I have Active and Non-Active Employees and I want to auto set the page to Active on load and set the select drop down to it.

**Comment by SteveyPugs** [comment]
Configure all 3 cases via js/css

- For the CSS I simply added my own CSS edits after yours by overriding certain classes within my controller.
- For the Filter I set the dropdown via jquery and injected a change event.

**Comment by wenzhixin** [comment]
So have you solved your problem?

**Comment by SteveyPugs** [comment]
With a few small minor hacks yes.

**Comment by wenzhixin** [comment]
Great!

**Comment by SteveyPugs** [comment]
It would be nice though if you could provide css classes / styles to headers (alone)

It would also be nice if you could auto set filter controls on init. Those sound like two features that might be useful.

**Comment by wenzhixin** [comment]
The first one you can define a class to your`th`.

The second one you can set `filterControl: true` http://issues.wenzhixin.net.cn/bootstrap-table/#extensions/filter-control.html

**Comment by SteveyPugs** [comment]
The first one I get. It's mostly what I did.

The second one you are not following. I know how to use filter control. The extension works fine. What I want to do use the extension with a predefined selection on the tables init. So let's say I have list of people and in the input filter control for first name I want to auto fill it with the word Doug to just start it a dougs. I don't want the filter to start blank. I want it pre filled. I achieved this with (in my case a select box) pre selecting a item in the select box then invoking the change method on that box to make the table filter via that control.

Best Regards,
Stephen Pugliese
